pkg/pmem-csi-driver/controllerserver-master.go

```diff
@@ -289,12 +289,14 @@ func (cs *masterController) DeleteVolume(ctx context.Context, req *csi.DeleteVol
 			conn, err := cs.rs.ConnectToNodeController(node)
 			if err != nil {
 				glog.Warningf("Failed to connect to node controller:%s, stale volume(%s) on %s should be cleaned manually", err.Error(), vol.id, node)
+				return nil, status.Error(codes.Internal, "Failed to connect to node "+node+": "+err.Error())
```

Comment:
You can remove the `glog.Warningf`. The gRPC logging will take care of logging the error. That's how it is done elsewhere, there's no reason why it should be done differently here.

Not only that, the warning text is also incorrect now.

Reply:
Done